nexus/client.go

```diff
@@ -121,6 +121,16 @@ func NewDefaultClient(baseURL string) *Client {
 	return NewClient(baseURL).Build()
 }
 
+// SetUsername sets the client username
+func (c *Client) SetUsername(username string) {
```

Comment:
Could you please add one more interface to set both? I think it will be the most basic use case since credentials would probably change username and password: `SetCredentials(username, passord string)` :D

Reply:
Yeah, I was debating myself about that one. I was wondering if we should leave up to the users to extend the API and provide only building blocks. But I suppose you're right, it's likely that the main use case for this is to change both at once. I'll change it in a bit :-)